### DISCUSSION TITLE
NAVYA Bus Integration in CARLA Simulator

### DISCUSSION THREAD

**Discussion by mave8533** [carla-simulator/carla#7724] [General]
Hello everyone,

I am currently working on a project where I need to integrate the NAVYA shuttle bus model into the CARLA simulator. Unfortunately, the NAVYA model is not available in CARLA's default vehicle catalog.

I would like to know if anyone has experience or resources for importing new vehicle models, specifically the NAVYA shuttle bus, into CARLA.
Any guidance, resources, or tips on how to accomplish this would be greatly appreciated. Thank you in advance for your help!

**Comment by ADASDevelopment** [comment]
If you have the model, you can make it.
https://www.youtube.com/watch?v=0F3ugwkISGk
